README.md

```diff
@@ -465,6 +591,8 @@ prefix `--terragrunt-`. The currently available options are:
   environment variable. The default path is `.terragrunt` in the current directory.
 * `--terragrunt-non-interactive`: Don't show interactive user prompts. This will default the answer for all prompts to 
   'yes'. Useful if you need to run Terragrunt in an automated setting (e.g. from a script).  
+* `--terragrunt-working-dir`: Set the directory where Terragrunt should execute the `terraform` command. Default is the
+  current working directory.
```

Comment:
This description is now a little ambiguous given the `spin-up` functionality. Is "where Terragrunt should execute the `terraform` command" the "root" folder, or each individual folder. I know it's root, but consider clarifying or rewording.